lib/scientist.rb

```diff
@@ -15,16 +18,34 @@ module Scientist
   #
   # Returns the calculated value of the control experiment, or raises if an
   # exception was raised.
-  def science(name, opts = {})
+  def self.run(name, opts = {})
     experiment = Experiment.new(name)
-    experiment.context(default_scientist_context)
 
     yield experiment
 
     test = opts[:run] if opts
     experiment.run(test)
   end
 
+  # Define and run a science experiment.
+  #
+  # name - a String name for this experiment.
+  # opts - optional hash with the the named test to run instead of "control",
+  #        :run is the only valid key.
+  #
+  # Yields an object which implements the Scientist::Experiment interface.
```

Comment:
Could you add a note here that `default_scientist_context` is applied to the experiment? It's already implied from the doc on `default_scientist_context`, but now that there's two ways of running an experiment I think it should be explicit.

Reply:
I clarified this and rebased all of the changes together in bd261c036858788873e056d26c7059150c202c33